.github/workflows/build-manylinux.yml

```diff
@@ -32,24 +32,23 @@ on:
 
 jobs:
   build:
+    name: ${{ matrix.image }} [${{ matrix.arch }}]
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false  # if a particular matrix build fails, don't skip the rest
       matrix:
         # Split job into many matrix builds, because GH actions provides 20
-        # concurrent builds on ubuntu. 10 are used here
+        # concurrent builds on ubuntu. 4 are used here
         include:
           # no pypy and cpython >= cp310 on manylinux1
-          - { image: manylinux1, arch: x86_64, pyversions: cp36-* cp37-* cp38-* cp39-* }
-          - { image: manylinux1, arch: i686, pyversions: cp36-* cp37-* cp38-* cp39-* }
+          - { image: manylinux1, arch: x86_64, pyversions: "cp3[6-9]-*" }
 
-          # no pypy 3.9 on manylinux2010
-          - { image: manylinux2010, arch: x86_64, pyversions: cp* pp37-* pp38-* }
-          - { image: manylinux2010, arch: i686, pyversions: cp* pp37-* pp38-* }
+          # no pypy >= 3.9 and cpython >= cp311 on manylinux2010
+          - { image: manylinux2010, arch: x86_64, pyversions: "cp3{[6-9],10}-* pp3{7,8}-*" }
 
-          # all cpython and pypy versions should support this
-          - { image: manylinux2014, arch: x86_64, pyversions: cp* pp* }
-          - { image: manylinux2014, arch: i686, pyversions: cp* pp* }
+          # all cpython and pypy versions should support this for now
+          - { image: manylinux2014, arch: x86_64, pyversions: "*" }
+          - { image: manylinux2014, arch: i686, pyversions: "*" }
```

Comment:
ALL?

You mean all valid versions from the perspective of cibuildwheel? What does that mean?

Reply:
All versions the specific cibuildwheel version supports

Reply:
So right now that happens to line up with what we support 3.6, but it won't always? I know that we have a pinned dependency version, but I'm still thinking about that.

Reply:
Till someone goes ahead and bumps the cibuildwheel version, nothing is going to change here. Even when cibuildwheel drops 3.6 in a version, we won't be affected till we bump it on our side (which is why pinned dependencies are ideal)

Also in this particular instance `cp* pp*` is basically equivalent to `*` but IMHO more readable as it is immediately clear that this tells cibuildwheel to use everything it's got

Reply:
Till someone goes ahead and bumps the cibuildwheel version, nothing is going to change here. Even when cibuildwheel drops 3.6 in a version, we won't be affected till we bump it on our side (which is why pinned dependencies are ideal)

Also in this particular instance `cp* pp*` is basically equivalent to `*` but IMHO more readable as it is immediately clear that this tells cibuildwheel to use everything it's got

Reply:
Leave a comment though just for future people?